docs/guides/administrator/gcp-setup.md

```diff
@@ -0,0 +1,162 @@
+# How to set up a GCP project for Opstrace
+
+This is a detailed walk-through showing how to set up a fresh GCP project for Opstrace.
+
+## Step 1: create a project
+
+Start by creating a fresh project with the [GCP UI](https://console.cloud.google.com/projectcreate).
+In addition to using a good project name (which you can change later), be sure to choose an expressive **project ID** on that screen (as the UI says: **it cannot be changed later**).
+
+
+**Note:** to prevent confusion, maybe first pick the project ID and then make the name match the ID.
+
+Take note of that project ID.
+We will use it down below.
+
+Next up, enable billing for that new project: link a GCP billing account to it, via [the GCP UI](https://console.cloud.google.com/billing/linkedaccount).
+
+**Note:** some of the next steps will not work if you don't link a billing account.
+
+## Step 2: set up the  `gcloud` CLI
+
+The `gcloud` command-line tool is a part of the Google Cloud SDK.
+You must [download and install that SDK](https://cloud.google.com/sdk/docs/install) on your system and initialize it before you can use the `gcloud` CLI.
+
+After installing `gcloud`, start the login process with `gcloud auth login`.
+Log in using the same GCP account that you created the project with above.
+
+Review `gcloud`'s authentication state:
+
+```bash
+gcloud auth list
+```
+
+The remaining steps assume that `gcloud` is now in fact logged in to a GCP account that has admin privileges for the newly created project.
+
+In the current terminal, set these two environment variables:
+
+```text
```

Comment:
A drive-by glance...  wondering why you switched to `text` from `bash`?

Reply:
In my experience across publishing platforms, `text` usually yields the more predictable syntax highlighting results when showing individual commands. `bash` syntax highlighting can sometimes be rather colorful/wrong for individual commands :) That's why I often prefer `text` when don't see the result yet. Depends on the syntax highlighting solution of course. Will have to get used to our solution here :).

Reply:
Understood.  Let's try to dial this in so we can have proper highlighting in our docs eventually.